target/site/jacoco/index.html

```diff
@@ -0,0 +1 @@
greenbar.gif" width="9" height="10" title="96" alt="96"/></td><td class="ctr2" id="c5">100 %</td><td class="bar" id="d6"/><td class="ctr2" id="e6">n/a</td><td class="ctr1" id="f5">0</td><td class="ctr2" id="g6">5</td><td class="ctr1" id="h5">0</td><td class="ctr2" id="i7">5</td><td class="ctr1" id="j5">0</td><td class="ctr2" id="k6">5</td><td class="ctr1" id="l5">0</td><td class="ctr2" id="m2">5</td></tr><tr><td id="a5"><a href="com.kenyajug.regression.resources/index.html" class="el_package">com.kenyajug.regression.resources</a></td><td class="bar" id="b6"><img src="jacoco-resources/greenbar.gif" width="6" height="10" title="69" alt="69"/></td><td class="ctr2" id="c6">100 %</td><td class="bar" id="d7"/><td class="ctr2" id="e7">n/a</td><td class="ctr1" id="f6">0</td><td class="ctr2" id="g7">5</td><td class="ctr1" id="h6">0</td><td class="ctr2" id="i8">5</td><td class="ctr1" id="j6">0</td><td class="ctr2" id="k7">5</td><td class="ctr1" id="l6">0</td><td class="ctr2" id="m3">5</td></tr><tr><td id="a0"><a href="com.kenyajug.regression/index.html" class="el_package">com.kenyajug.regression</a></td><td class="bar" id="b7"><img src="jacoco-resources/greenbar.gif" width="5" height="10" title="62" alt="62"/></td><td class="ctr2" id="c7">100 %</td><td class="bar" id="d4"><img src="jacoco-resources/greenbar.gif" width="4" height="10" title="2" alt="2"/></td><td class="ctr2" id="e4">100 %</td><td class="ctr1" id="f7">0</td><td class="ctr2" id="g5">9</td><td class="ctr1" id="h7">0</td><td class="ctr2" id="i5">17</td><td class="ctr1" id="j7">0</td><td class="ctr2" id="k5">8</td><td class="ctr1" id="l7">0</td><td class="ctr2" id="m7">3</td></tr><tr><td id="a6"><a href="com.kenyajug.regression.scheduled/index.html" class="el_package">com.kenyajug.regression.scheduled</a></td><td class="bar" id="b8"><img src="jacoco-resources/greenbar.gif" width="1" height="10" title="15" alt="15"/></td><td class="ctr2" id="c8">100 %</td><td class="bar" id="d8"/><td class="ctr2" id="e8">n/a</td><td class="ctr1" id="f8">0</td><td class="ctr2" id="g8">3</td><td class="ctr1" id="h8">0</td><td class="ctr2" id="i6">6</td><td class="ctr1" id="j8">0</td><td class="ctr2" id="k8">3</td><td class="ctr1" id="l8">0</td><td class="ctr2" id="m8">1</td></tr><tr><td id="a3"><a href="com.kenyajug.regression.models/index.html" class="el_package">com.kenyajug.regression.models</a></td><td class="bar" id="b9"><img src="jacoco-resources/greenbar.gif" width="1" height="10" title="12" alt="12"/></td><td class="ctr2" id="c9">100 %</td><td class="bar" id="d9"/><td class="ctr2" id="e9">n/a</td><td class="ctr1" id="f9">0</td><td class="ctr2" id="g9">1</td><td class="ctr1" id="h9">0</td><td class="ctr2" id="i9">1</td><td class="ctr1" id="j9">0</td><td class="ctr2" id="k9">1</td><td class="ctr1" id="l9">0</td><td class="ctr2" id="m9">1</td></tr></tbody></table><div class="footer"><span class="right">Created with <a href="http://www.jacoco.org/jacoco">JaCoCo</a> 0.8.12.202403310830</span></div></body></html>
```

Comment:
The test coverage is automatically written to the project readme, you should not create this file

Once you run `mvn verify` the readme file should automatically update

Consider deleting this file